stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -247,6 +254,18 @@ private void updateIcon(@NonNull @Card.CardBrand String brand) {
         }
     }
 
+    private void updateIconCvc(@NonNull @Card.CardBrand String brand, boolean isEntering) {
```

Comment:
I included the "focus" information here (rather than just dealing with it in the listener) because we are planning to add a "card flip" animation to the transition, which would only be done if we were *not* in AmEx mode, so I'd need to know a little bit of information before just calling updateIcon with the brand.